src/buildParameters.ts

```diff
@@ -71,100 +59,97 @@ export function buildParameters(uri: Uri): EvaluationParameters {
   return evaluationParams;
 }
 
-function getFhirVersion(): string | null {
-  const fhirVersionRegex = /using (FHIR|"FHIR") version '(\d(.|\d)*)'/;
-  const matches = window.activeTextEditor!.document.getText().match(fhirVersionRegex);
-  if (matches && matches.length > 2) {
-    const version = matches[2];
-    if (version.startsWith('2')) {
-      return 'DSTU2';
-    } else if (version.startsWith('3')) {
-      return 'DSTU3';
-    } else if (version.startsWith('4')) {
-      return 'R4';
-    } else if (version.startsWith('5')) {
-      return 'R5';
-    }
-  }
+function getCqlCommandArgs(
```

Comment:
This is a *lot* of parameters, could we pass in an parameter object instead? Something like:

```typescript
function getCqlCommandArgs({
  fhirVersion,
  optionsPath,
  libraryDirectory,
  libraryName,
  expression,
  terminologyPath,
  connection,
  contexts,
  measurementPeriod,
}): {
  fhirVersion: string,
  optionsPath: Uri,
  libraryDirectory: Uri,
  libraryName: string,
  expression: string | undefined,
  terminologyPath: Uri,
  connection: Connection | undefined,
  contexts: Map<string, Context>,
  measurementPeriod: string,
} 
```